pkg/object/httpserver/mux.go

```diff
@@ -44,6 +49,10 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+const (
+	defaultAccessLogFormat = "$Time $RemoteAddr $RealIP $Method $URI $Proto $StatusCode $Duration $ReqSize $RespSize ReqHeaders:{$ReqHeaders} RespHeaders:{$RespHeaders} Tags:{$Tags}"
```

Comment:
better keep this default format to the same format before this PR, some external tools may rely on the format for analysis, and we will break them if the format changes.